### PR TITLE
Add map equipments

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 1]
```diff
@@ -1027,4 +1052,28 @@ public List<BusbarSectionMapData> getVoltageLevelBusbarSections(UUID networkUuid
...
+        Network network = getNetwork(networkUuid, substationsId == null ? PreloadingStrategy.COLLECTION : PreloadingStrategy.NONE, variantId);
+
+        if (substationsId == null) {
+            return MapEquipmentsData.builder()
+                    .lines(network.getLineStream().map(NetworkMapService::toBasicMapData).collect(Collectors.toList()))
+                    .substations(network.getSubstationStream().map(NetworkMapService::toMapData).collect(Collectors.toList()))
+                    .build();
+        }
+
+        Set<LineMapData> lines = new LinkedHashSet<>();
```
[the remark this answers is not in the document]
Comment (Meklo) [reply src/main/java/org/gridsuite/network/map/NetworkMapService.java]: Done

**Review by niconoir — APPROVED** [review]

File: src/main/java/org/gridsuite/network/map/model/MapEquipmentsData.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.gridsuite.network.map.model;
```
Comment (laurgarn) [inline src/main/java/org/gridsuite/network/map/model/MapEquipmentsData.java:1]: Header is missing here, no ?

Comment (Meklo) [reply src/main/java/org/gridsuite/network/map/model/MapEquipmentsData.java:7]: Indeed, I've added it

**Review by niconoir — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Code Review OK
Test OK
Console Warning Check OK

**Pull request merged**